Guide/Greedy/Interval Scheduling/README.md

```diff
@@ -0,0 +1,181 @@
+# Interval Scheduling
+
+This problem is discussed in [Dr. Gelfond Applied Algorithms](http://redwood.cs.ttu.edu/~mgelfond/FALL-2012/slides.pdf) and in [Wikipedia](https://en.wikipedia.org/wiki/Interval_scheduling)
+ 
+
+Category: Greedy
+
+Difficulty: N/A
```

Comment:
I might actually set a difficulty yourself. Just to give people an idea of what they are getting into